src/target/riscv/program.c

```diff
@@ -182,9 +182,8 @@ int riscv_program_fence_rw_rw(struct riscv_program *p)
 int riscv_program_ebreak(struct riscv_program *p)
 {
 	struct target *target = p->target;
-	RISCV_INFO(r);
-	if (p->instruction_count == riscv_progbuf_size(p->target) &&
-			r->impebreak) {
+	if (p->instruction_count == riscv_progbuf_size(target) &&
+			riscv_info(target)->get_impebreak(target)) {
```

Comment:
(same as in previous comment)

I'd use RISCV_INFO() here - I'd prefer consistency over a saved line of code.

```suggestion
	RISCV_INFO(r);
	if (p->instruction_count == riscv_progbuf_size(target) &&
			r->get_impebreak(target)) {
```

Reply:
Addressed